src/aura/CRLP_Rollup/CRLP_RollupHelper.js

```diff
@@ -912,27 +912,37 @@
     setDefaultAmount: function(cmp, labels, amountObjectName) {
         var summaryFieldType = this.retrieveFieldType(cmp, cmp.get("v.activeRollup.summaryField"), cmp.get("v.summaryFields"));
 
-        if (summaryFieldType !== 'PERCENT') {
+        // only assume the amount field if the summary field is a number or currency.
+        // if the summary field is a percent, the admin has to explicitly select an amount field
+        var setAmountField = (summaryFieldType !== 'PERCENT');
 
-            var amountFieldName;
-            var amountFields = cmp.get("v.amountFields");
+        var amountFieldName;
+        var amountFields = cmp.get("v.amountFields");
 
-            if (amountObjectName === labels.objectPayment) {
+        if (amountObjectName === labels.objectPayment) {
+            if(setAmountField) {
                 amountFieldName = labels.objectPayment + ' npe01__Payment_Amount__c';
-                cmp.set("v.activeRollup.detailObject", labels.objectPayment);
-            } else if (amountObjectName === labels.objectAllocation) {
+            }
+            cmp.set("v.activeRollup.detailObject", labels.objectPayment);
+        } else if (amountObjectName === labels.objectAllocation) {
+            if(setAmountField) {
                 amountFieldName = labels.objectAllocation + ' ' + labels.namespacePrefix + 'Amount__c';
-                cmp.set("v.activeRollup.detailObject", labels.objectAllocation);
-            } else if (amountObjectName === labels.objectPartialSoftCredit) {
+            }
+            cmp.set("v.activeRollup.detailObject", labels.objectAllocation);
+        } else if (amountObjectName === labels.objectPartialSoftCredit) {
+            if(setAmountField) {
                 amountFieldName = labels.objectPartialSoftCredit + ' ' + labels.namespacePrefix + 'Amount__c';
-                cmp.set("v.activeRollup.detailObject", labels.objectPartialSoftCredit);
-            } else {
+            }
+            cmp.set("v.activeRollup.detailObject", labels.objectPartialSoftCredit);
+        } else {
+            if(setAmountField) {
                 amountFieldName = labels.objectOpportunity + ' Amount';
-                cmp.set("v.activeRollup.detailObject", labels.objectOpportunity);
             }
-            cmp.set("v.activeRollup.amountField", amountFieldName);
-            cmp.set("v.activeRollup.amountFieldLabel", this.retrieveFieldLabel(amountFieldName, amountFields));
+            cmp.set("v.activeRollup.detailObject", labels.objectOpportunity);
         }
+        cmp.set("v.activeRollup.amountField", amountFieldName);
+        cmp.set("v.activeRollup.amountFieldLabel", this.retrieveFieldLabel(amountFieldName, amountFields));
```

Comment:
Is this a conscious choice to have amountFieldName set as undefined for percents? Also, is setting the amountFieldLabel necessary for percents too?